libs/sdk-backend-tiger/package.json

```diff
@@ -55,6 +55,7 @@
         "date-fns": "^2.8.1",
         "http-status-codes": "^1.3.0",
         "lodash": "^4.17.19",
+        "moment": "^2.24.0",
```

Comment:
We already have date-fns in this package, can we use it onstead of moment?

Reply:
Great point! Rewritten to date-fns in https://github.com/gooddata/gooddata-ui-sdk/pull/1678/commits/77e716babf440a309075cd0294853296206cae71 and also noticed that the snapshots had hardcoded date for the time they were created (https://github.com/gooddata/gooddata-ui-sdk/blob/77e716babf440a309075cd0294853296206cae71/libs/sdk-backend-tiger/src/backend/workspace/attributes/elements/tests/__snapshots__/date.test.ts.snap) so now the methods accept current date from top https://github.com/gooddata/gooddata-ui-sdk/pull/1678/commits/77e716babf440a309075cd0294853296206cae71#diff-94d7f29b9b443070696250236b16dc69094b46e3200a0f838f3f355798c97394R24